app/classifier/tasks/generic.cjsx

```diff
@@ -48,6 +48,22 @@ module.exports = React.createClass
             </small>
           </p>
         </div>}
+
+      {if @props.toggleHideMarksBefore
+        <div>
+          <p>
+            <small>
+              <strong>
+                <label className="hide-previous-marks-toggle">
+                  <input type="checkbox" onClick={@props.toggleHideMarksBefore}>
+                    Hide previous marks
+                  </input>
+                </label>
```

Comment:
Thanks, @simensta. I was wondering why I kept getting that warning! I'll take care of that when I get a chance